spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/context/properties/AwsCredentialsProperties.java

```diff
@@ -53,6 +54,21 @@
 	 */
 	private String profilePath;
 
+	/**
+	 * The arn of the role to be assumed.
+	 */
+	private String roleArn;
```

Comment:
What do you think about moving all new properties to static inner class `StsCredentialsProperties` so that they are configured via properties: `cloud.aws.credentials.sts.roleArn` etc.?

Reply:
Done